src/Components/Blazor/Build/test/BuildIntegrationTests/PublishIntegrationTest.cs

```diff
@@ -155,31 +155,23 @@ public async Task Publish_HostedApp_Works()
             // Make sure the main project exists
             Assert.FileExists(result, publishDirectory, "blazorhosted.dll");
 
-            var blazorPublishDirectory = Path.Combine(publishDirectory, "standalone");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "blazor.boot.json");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "blazor.webassembly.js");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "wasm", "mono.wasm");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "wasm", "mono.js");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "_bin", "standalone.dll");
-            Assert.FileExists(result, blazorPublishDirectory, "dist", "_framework", "_bin", "Microsoft.Extensions.Logging.Abstractions.dll"); // Verify dependencies are part of the output.
+            var blazorPublishDirectory = Path.Combine(publishDirectory, "wwwroot");
```

Comment:
@pranavkm Not sure what you mean. I believe that we wanted to keep the Blazor publish output in a separate directory for standalone applications.

So standalone ends up in publish/AppName/dist and hosted ends up in wwwroot/<<SWABasePath>> (which defaults to "/")

Reply:
They don't, as far as I understand it is intentional that we leave standalone apps at `/publish/<app-name>/dist`. Otherwise I would be avoiding all these shenanigans

Reply:
See my question above. Do we know why we do that?

Reply:
@pranavkm Are you asking why the standalone publish output uses the `/publish/<appname>/dist` structure?

If so it's because that was the safest thing to do at the time we implemented it for Blazor 0.1.0. The SDK contains a lot of nontrivial logic that puts stuff into the root of `/publish`, and we would need to track down each such thing, find out how to stop it doing that, and make sure it's all safe to do.

I fully agree it would be much better if we only put what is now in `/dist` directly into the root of `/publish`, and nothing else. It would be a really big improvement. It just requires somebody to dig into the SDK's publish targets in more detail and be sure we know how to do that. Maybe it's quite easy, I don't know :)

Reply:
We will now publish to /publish/wwwroot for standalone apps